File: expo-app/src/lib/etfMeta.ts
export interface EtfMeta {
  name: string;
  color: string;
}

export const ETF_ORDER = ['0050', '00631L', 'QQQ', 'QQQM'] as const;
export type EtfSymbol = (typeof ETF_ORDER)[number];

export const ETF_META: Record<EtfSymbol, EtfMeta> = {
  '0050':   { name: '元大台灣50',    color: '#F59E0B' },
  '00631L': { name: '元大台灣50正2', color: '#34D399' },
  'QQQ':    { name: 'Invesco QQQ',  color: '#4C9BE8' },
  'QQQM':   { name: 'QQQM',         color: '#A78BFA' },
};

export const PRESETS: [string, number][] = [
  ['3K', 3000],
  ['5K', 5000],
  ['1W', 10000],
  ['2W', 20000],
  ['3W', 30000],
];

File: ios-app/ETFCalc/Models/ETFMeta.swift
import SwiftUI

struct ETFMeta {
    let name: String
    let color: Color
}

let ETF_ORDER: [String] = ["0050", "00631L", "QQQ", "QQQM"]

let ETF_META: [String: ETFMeta] = [
    "0050":   ETFMeta(name: "元大台灣50",    color: Color(red: 0.96, green: 0.62, blue: 0.04)),
    "00631L": ETFMeta(name: "元大台灣50正2", color: Color(red: 0.20, green: 0.83, blue: 0.60)),
    "QQQ":    ETFMeta(name: "Invesco QQQ",  color: Color(red: 0.30, green: 0.61, blue: 0.91)),
    "QQQM":   ETFMeta(name: "QQQM",         color: Color(red: 0.65, green: 0.55, blue: 0.98)),
]

let PRESETS: [(String, Int)] = [
    ("3K",  3_000),
    ("5K",  5_000),
    ("1W",  10_000),
    ("2W",  20_000),
    ("3W",  30_000),
]
